src/s3.js

```diff
@@ -128,3 +146,12 @@ const whenGotS3Object = s3 => params => {
     })
   })
 }
+
+const whenGotS3ObjectStream = s3 => params => {
+  return new Promise((resolve, reject) => {
+    var fileStream = fs.createWriteStream('./temp/file.csv')
```

Comment:
I think this line is unnecessary and was throwing an error until I commented it out